index.html

```diff
@@ -1677,19 +1673,15 @@
                 </li>
               </ol>
             </li>
-            <li>If the <a>presentation availability promise</a> for
-            <code>presentationRequest</code> is not <code>null</code>, return
-            the <code>presentationRequest</code> object's <a>presentation
-            availability promise</a> and abort these steps.
+            <li>If there is an unsettled <a>Promise</a> from a previous call to
```

Comment:
If the returned Promise is settled, it will generate a new Promise, but will return the same `PresentationAvailability` object if it still exists "If the presentation display availability for presentationRequest is not null, then Resolve P with the request's presentation display availability" (step 6).

This works as **I** would like it to work but I am unclear how **we** would like it to. As noted in:
> I think the intent with getAvailability is to return the same Promise as long as it is unsettled. To be confirmed with @schien (also raised in #388 (comment)). If the goal is rather to return the same Promise as long as the code holds a reference to a Promise previously returned by that function, the suggestion below does not work.
> https://github.com/w3c/presentation-api/pull/390#issuecomment-265115507

If I drop "unsettled", the same Promise will be returned as long as it has not been garbage collected. Is it the intent?